### ISSUE TITLE
[Graph Libraries] windows 64 bit version

### ISSUE THREAD

**Issue by GadgetSteve** [ellson/graphviz#597]
Ported Issue from Mantis
Original ID: 1995
Reported By: Kevin2

SEVERITY: MINOR
Submitted: 2010-11-22 22:52:26

OS: OTHER-WINDOWS-2008R2

DESCRIPTION
-----------

Would it be possible to get a 64 bit version compiled?  I'm trying to use the gvc.dll inside a 64bit application and get an error because the 64bit process cannot use the 32 bit dll.

**Comment by Chiel92** [comment]
Yes, this would be great. Is there progression since this issue was created?
